Externals/Vulkan/Include/vulkan/vulkan_core.h

```diff
@@ -47,7 +47,11 @@ extern "C" {
 #define VK_HEADER_VERSION 121
 
 
+#if defined(__LP64__) || defined(_WIN64) || (defined(__x86_64__) && !defined(__ILP32__) ) || defined(_M_X64) || defined(__ia64) || defined (_M_IA64) || defined(__aarch64__) || defined(__powerpc64__)
```

Comment:
Is this needed? 0 should work for both u64s and pointer types for comparison.

Reply:
Given this is modifying an externel header, this can also break the build if the headers are ever updated in the future (as new versions of the header won't contain this change).

Reply:
Right, that's probably not a good idea to have a local version of external code. I'll push an updated version of the patch not modifying Externals/.